.yamllint

```diff
@@ -0,0 +1,3 @@
+---
+rules:
+  truthy: disable  # The current version of yaml, checking keys.
```

Comment:
Why you need this?

Reply:
The current version of yaml, checking keys. While in travis we use `on`

Reply:
https://github.com/adrienverge/yamllint/blob/master/yamllint/rules/truthy.py

Reply:
"truthy value is not quoted"

easy solution -- quote it.

or use alternative travis syntax to achieve the same effect.

Reply:
It might also be a bug in yamllint, in which case you should look at yamllint to see if there is an existing issue, and create an issue if it doesnt already exist.

Reply:
It is upstream bugs.